R/existsBackgroundVariables.r

```diff
@@ -15,6 +15,7 @@ existsBackgroundVariables <- function(dat, variable, warnIfMissing = FALSE )  {
                   			varColumn <- match(variable, colnames(dat))
                  	   }
                      if(is.numeric(variable))   {
+                        if ( !all(round(variable) == variable)) {stop(paste0("If provided as column number, variable '",variable,"' must be integer."))}

```

Comment:
`paste0()` could simply be omitted here (`stop()` pastes automatically).

Reply:
will be done